mne_bids/write.py

```diff
@@ -951,10 +961,6 @@ def write_raw_bids(raw, bids_path, bids_root, events_data=None,
                        "raw.times should not have changed since reading"
                        " in from the file. It may have been cropped.")
 
-    # convert to BIDS Path
-    if isinstance(bids_path, str):
-        bids_path = _convert_str_to_bids_path(bids_path)
```

Comment:
is _convert_str_to_bids_path still used somewhere?

Reply:
No, at least not in this fork ;)